jupyter_scheduler/exceptions.py

```diff
@@ -1,3 +1,11 @@
+class SchedulerError(Exception):
```

Comment:
Will this enable a backend to raise a `SchedulerError` with a custom message and have that sent back to the frontend to display the error message in the create form banner?

Reply:
Correct.